src/transform/CollapseTable.js

```diff
@@ -218,9 +218,27 @@ const collapseTables = (document, content, pageTitle, isMainPage, infoboxTitle,
   }
 }
 
+/**
+ * @param  {?Element} element
+ * @return {void}
```

Comment:
Wondering how we want to deal with things like a comment reminding us where issues can be seen... i.e. for this one:
> See first reference link - the `[A]` in `enwiki > Airplane` article from ~June 2016.